docs/extend/modular-resources/_index.md

```diff
@@ -110,9 +110,12 @@ For example:
 {{< tabs name="Base Model Modules" >}}
 {{% tab name="Go" %}}
 
-This example module code is adapted from the full demo module available on the [Viam GitHub](https://github.com/viamrobotics/rdk/blob/main/examples/customresources/models/mybase/mybase.go), and creates a singular modular resource implementing Viam's built-in Base API (rdk:service:base).
+This example custom model code (<file>mybase.go</file>) and module (<file>mymodule.go</file>) is adapted from the full demo module available on the [Viam GitHub](https://github.com/viamrobotics/rdk/blob/main/examples/customresources/models/mybase/mybase.go), and creates a singular modular resource implementing Viam's built-in Base API (rdk:service:base).
```

Comment:
I recognize that my comment is about a part of the line you didn't edit, but what does "singular" mean in this context? It's not the only one, it's not the best one, it's not a unique one. Could we remove that word and just say it "creates a modular resource"?